flocker/node/agents/cinder.py

```diff
@@ -852,19 +858,19 @@ def get_keystone_session(**config):
         )
 
 
-def get_cinder_v1_client(session, region):
+def get_cinder_client(session, region, version):
```

Comment:
Do we expect that get_cinder_v1_client is internal-only? The path to it is fully public, so anyone using flocker-as-a-library can get burned by that. Otherwise we should support a get_cinder_v1 -> get_cinder shim?